build/configs/artik053/STDK/defconfig

```diff
@@ -258,7 +258,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="16,48,192,32,512,2400,1536,1536,1000,400,8,512,"
-CONFIG_FLASH_PART_TYPE="none,ftl,none,none,none,none,none,ftl,smartfs,romfs,config,none,"
+CONFIG_FLASH_PART_TYPE="none,ftl,none,none,none,none,none,ftl,jSmartFs,romfs,config,none,"
```

Comment:
Do we really need to name it as `jSmartFs` here as well? 
@sangam-swami @amogh-samsung @Taejun-Kwon Could you give me your opinion?

Reply:
This change is intended.
Changes hall not be made to function and fie names, etc
But we intend to change the partition name from smartfs to jSmartFs for outright differentiation.

Reply:
This is the part of user input. I don't agree something could be complex here. Simple is best to user I think. (I want just `fs`.)
But this change needs to distinguish upper and lower case and needs `j`. 
I don't think user wants to know it's jsmartfs and smartfs. 
I agree adding it in the file system itself. But not here.
If we have two options, using jsmartfs and smartfs, then it's ok. But it's not.
And this is not a fs config.
@amogh-samsung @sangam-swami @mkrunal @Taejun-Kwon Please let me know your opinion.

Reply:
@amogh-samsung Sorry for late reply.

Reply:
I acknowledge your point.
We will keep it according to the general consensus, whatever everyone thinks is best.
Waiting for others' opinions too.

Reply:
We have updated only file system name and not changing from smartfs to jSmarrtFs here for now.